install.rdf

```diff
@@ -65,32 +72,8 @@
         <em:targetApplication>
           <Description>
             <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id> <!-- Firefox -->
-            <em:minVersion>17</em:minVersion>
-            <em:maxVersion>51.*</em:maxVersion>
-          </Description>
-        </em:targetApplication>
-
-        <em:targetApplication>
-          <Description>
-            <em:id>{8de7fcbb-c55c-4fbe-bfc5-fc555c87dbc4}</em:id> <!-- Palemoon -->
-            <em:minVersion>3.6</em:minVersion>
-            <em:maxVersion>25.*</em:maxVersion>
-          </Description>
-        </em:targetApplication>
-
-        <em:targetApplication>
-          <Description>
-            <em:id>{3550f703-e582-4d05-9a08-453d09bdfdc6}</em:id> <!-- Thunderbird -->
-            <em:minVersion>10.0</em:minVersion>
-            <em:maxVersion>34.*</em:maxVersion>
-          </Description>
-        </em:targetApplication>
-
-        <em:targetApplication>
-          <Description>
-            <em:id>{92650c4d-4b8e-4d2a-b7eb-24ecf4f6b63a}</em:id> <!-- Seamonkey -->
-            <em:minVersion>2.0</em:minVersion>
-            <em:maxVersion>2.32.*</em:maxVersion>
+            <em:minVersion>51.0</em:minVersion>

```

Comment:
Min version is 52 and the current version for Firefox Dev Edition is 54.0a2. We may want to use the latest as the max.